lms/templates/courseware/courses.html

```diff
@@ -56,12 +56,19 @@
         % endif
 
         <div class="courses${'' if course_discovery_enabled else ' no-course-discovery'}" role="region" aria-label="${_('List of Courses')}">
+          <ul class="courses-listing bundle-list">
```

Comment:
Add corresponding changes to mitxpro theme whitelabel/journals branch please:
https://github.com/edx/edx-themes/blob/whitelabel/journals/edx-platform/mitxpro.mit.edu/lms/templates/courseware/courses.html#L40